cmd/remotefs/main.go

```diff
@@ -94,13 +98,13 @@ func main() {
 	// Decode information
 	bytes, err := base64.StdEncoding.DecodeString(*base64string)
 	if err != nil {
-		logrus.Fatalf("Failed to decode base64: %s", err.Error())
+		logrus.Fatalf("Failed to decode base64: %s\n"+ERROR_STRING, err.Error())

```

Comment:
spaces around `+` go back and forth between with and without

Reply:
Yeah it's been some weird auto fix in vscode that I assumed was happening everywhere within the files